### PR TITLE
Add incubation disclaimer

### PR DESCRIPTION
**Pull request by wikier**
As required by http://incubator.apache.org/guides/branding.html#disclaimers

### CODE REVIEW

**Comment by dhalperi** [comment]
R: @jbonofre @evilsoapbox

**Comment by evilsoapbox** [comment]
Excellent catch. LGTM. I am merge ASAP (getting ready for Beam talk today at Hadoop Summit :)

File: index.md [hunk 1]
```diff
@@ -90,6 +90,12 @@ Interested in working with Apache Beam? Great! Here's how to get started:
...
-available under the Apache v2 license.
+## Apache Incubation Disclaimer
+
+Apache Beam is an effort undergoing incubation at [The Apache Software Foundation (ASF)](http://www.apache.org) 
+sponsored by the Apache Incubator PMC. Incubation is required of all newly accepted projects until a further review
+indicates that the infrastructure, communications, and decision making process have stabilized in a manner consistent 
+with other successful ASF projects. While incubation status is not necessarily a reflection of the completeness or
+stability of the code, it does indicate that the project has yet to be fully endorsed by the ASF.
+
+Apache Bean is available under [Apache License, version 2.0](http://www.apache.org/licenses/LICENSE-2.0).
```
Comment (dhalperi) [inline index.md]: Beam

**Comment by jbonofre** [comment]
LGTM

File: index.md (hunk 1 above)
Comment (evilsoapbox) [inline index.md]: Line 101 Bean -> Beam

**Comment by evilsoapbox** [comment]
LGTM - going to merge.

**Pull request merged**